src/actions/config.js

```diff
@@ -108,19 +124,12 @@ export function loadConfig() {
 
     try {
       const preloadedConfig = getState().config;
-      const response = await fetch('config.yml', { credentials: 'same-origin' })
-      const requestSuccess = response.status === 200;
-
-      if (!preloadedConfig && !requestSuccess) {
-        throw new Error(`Failed to load config.yml (${ response.status })`);
-      }
-
-      const loadedConfig = parseConfig(requestSuccess ? await response.text() : '');
+      const loadedConfig = await getConfig('config.yml', preloadedConfig && preloadedConfig.size > 1);
```

Comment:
In testing, preloadedConfig has an iterable object map with one key `isFetching` if the preload did not have a JSON preload using `init()`.